build.gradle

```diff
@@ -0,0 +1,57 @@
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
+    jcenter()
+    mavenLocal()
+  }
+
+  dependencies {
+    classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.8.1"
+  }
+}
+
+group "com.trusona.integration"
+version "1.3.0-SNAPSHOT"
+
+apply plugin: "java"
+apply plugin: "groovy"
+apply plugin: "idea"
+apply plugin: "maven"
+apply plugin: "jacoco"
+apply from: "publishing.gradle"
+apply from: "build-extras.gradle"
```

Comment:
Do we need to use the node plugin? How does Travis get Node/npm?

Reply:
I added a node/npm plugin that does the same thing the maven plugin was doing i.e. installing a specific version of node and npm.